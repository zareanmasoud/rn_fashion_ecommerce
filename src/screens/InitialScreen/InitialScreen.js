import React from 'react';
import {View} from 'react-native';
// eslint-disable-next-line import/no-extraneous-dependencies
import Loader from 'lib/components/Loader';
import styles from './styles';
import {HOME_SCREEN} from '../routes/constants';

class InitialScreen extends React.Component {
  constructor(props) {
    super(props);

    // temporarily
    const {navigation} = this.props;
    setTimeout(() => navigation.navigate(HOME_SCREEN), 1000);
  }

  render() {
    return (
      <View style={styles.container}>
        <Loader />
      </View>
    );
  }
}

export default InitialScreen;
