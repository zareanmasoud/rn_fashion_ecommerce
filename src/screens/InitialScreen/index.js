import InitialScreen from './InitialScreen';

export {InitialScreen as default};
