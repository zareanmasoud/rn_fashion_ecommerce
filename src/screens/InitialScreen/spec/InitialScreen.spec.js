import React from 'react';
import {shallow} from 'enzyme';
import InitialScreen from '../InitialScreen';

describe('InitialScreen', () => {
  describe('Rendering', () => {
    it('should match to snapshot', () => {
      const wrapper = shallow(<InitialScreen />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
