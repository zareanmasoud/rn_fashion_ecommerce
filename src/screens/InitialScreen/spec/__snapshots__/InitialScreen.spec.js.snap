// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InitialScreen Rendering should match to snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Loader
    size="small"
  />
</View>
`;
